app/src/main/java/org/mozilla/fenix/addons/NotYetSupportedAddonFragment.kt

```diff
@@ -52,10 +56,16 @@ class NotYetSupportedAddonFragment :
         showToolbar(getString(R.string.mozac_feature_addons_unsupported_section))
     }
 
+    override fun onBackPressed(): Boolean = false
```

Comment:
@boek `onBackPressed() = false` means we're letting `UserInteractionHandler` handle the back press, correct?

Reply:
@psymoon Yes, but since we aren't overloading this it might make more sense to just call `navController.popBackStack()`